webroot/js/modules/Toolbar.js

```diff
@@ -152,7 +152,14 @@ export default class Toolbar {
       // This initializes the panel specific JS logic (if there is any)
       document.dispatchEvent(new CustomEvent('initPanel', { detail: `panel${panelType}` }));
       that.bindDebugBlock();
-    });
+    })
+      .fail(() => {
+        clearTimeout(timer);
+        $(`<iframe src="${url}" width="100%" height="100%"></iframe>`).appendTo(contentArea);
```

Comment:
Why use another iframe instead of updating the content of `contentArea`?

Reply:
I first wasn't able to get that to work for some reason but you are right, we don't need another iframe here.